app/src/main/java/com/macbitsgoa/ard/activities/BaseActivity.java

```diff
@@ -65,11 +67,16 @@ public void showToast(final String message) {
      *
      * @param message   Message that the toast must show.
      * @param toastType Duration for which the toast must be visible.
-     *
+     */
     public void showToast(final String message,
                           @IntRange(from = Toast.LENGTH_SHORT,
                                   to = Toast.LENGTH_LONG) final int toastType) {
-        Toast.makeText(this, message, toastType).show();
+        runOnUiThread(new Runnable() {
+            @Override
+            public void run() {
+                Toast.makeText(getApplicationContext(), message, toastType).show();
```

Comment:
This was a workaround for `java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()` error which happened at (`androidTest`) testing. But, I can avoid it. So, yeah, runnable not needed.